client/my-sites/stats/wordads/highlights-section.jsx

```diff
@@ -140,15 +139,45 @@ function HighlightsListing( props ) {
 	);
 }
 
+function HighlightsListingMobile( { highlights } ) {
+	// Convert the highlights data for the MobileHighlightCardListing component.
+	// Use preformattedValue property as an override to the count.
+	// Send the raw SVG icon and zero out the count.
+	const mobileHighlights = [
```

Comment:
I'd prefer we use a loop to handle the conversion here. Also any particular reason we are not using `highlight.icon` here?

Reply:
Per the comment, we want the raw SVG element. The data as provided in `highlight.icon` is actually an `Icon` component which isn't what we want here.

Reply:
Maybe adding the SVG element to `useHighlights` is a better approach here. Then we can update the mobileHighlights setup to use a loop. I'll give this a look too.